extensions/wikia/VideoHandlers/VideoFileUploader.class.php

```diff
@@ -134,7 +135,21 @@ public function upload( &$oTitle ) {
 			// @TODO
 			// if video already exists make sure that we are in fact changing something
 			// before generating upload (for now this only works for edits)
-			$article = Article::newFromID( $oTitle->getArticleID() );
+			$articleId = $oTitle->getArticleID();
+			$article = Article::newFromID( $articleId );
+			//In case Article is null
+			//Log more info and throw an exception
+			if ( !( is_null( $article ) ) ) {
```

Comment:
This states the opposite to the condition in the comment, shouldn't it be `if ( is_null( $article ) ) {…` (if article is NULL → throw an exception)?